components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -115,8 +115,8 @@ Vagrant.configure(2) do |config|
     echo "Configured cluster admin user: admin, Password: admin"
     echo
     echo "To use OpenShift CLI, run:"
-    echo "$ vagrant ssh"
-    echo "$ oc login"
+    echo "vagrant service-manager install-cli openshift"
```

Comment:
@budhrg I am not sure if `If you don't have OpenShift CLI, oc, install it as:` is better than `To download and install OC binary, run:`.  I think we should ask others.  @Preeticp @thatdocslady comments?